.evergreen/config.yml

```diff
@@ -1003,8 +991,6 @@ buildvariants:
   - build-and-test-and-upload
   - build-and-test-shared-bson
   - build-and-test-asan
-  - build-and-test-node
```

Comment:
I think it's nice to have task history, even with all the changes we're making. I _think_ this breaks that, maybe leave in build-and-test-node, and instead make the scripting no-op for the time being.